lib/Pipfile

```diff
@@ -9,7 +9,7 @@ black = "==21.6b0"
 boto3 = "*"
 botocore = ">=1.13.44"
 hypothesis = ">=6.17.4"
-mypy = ">=0.910"
+mypy = ">=0.910, <0.930"
```

Comment:
I don't think we should do this (in fact, if anything, we should pin `mypy = ">=0.930"`)

Reply:
That's was just a test if my build or mypy is broken. And as it turns out, mypy `0.930` (which came out few hours ago) breaks our build pipelines :(

Reply:
See the build logs here: https://app.circleci.com/pipelines/github/streamlit/streamlit/10654/workflows/9f4fcc98-06ec-474d-b7b8-84907ef9c7ca/jobs/34844/parallel-runs/0/steps/0-123

Reply:
I think it may just be necessary to pull in the latest changes from `develop`, since I believe https://github.com/streamlit/streamlit/pull/4191 fixed this

Reply:
I will try that. However, the builds went fine yesterday and those issues only appeared after mypy 0.930 got released. So, I'm assuming there are additional issues 😬

Reply:
Actually, the fixes from https://github.com/streamlit/streamlit/pull/4191 are already included in this branch.

Reply:
Ah, #4191 fixed the issues that appeared with mypy 0.920

I didn't realize this was a new set of issues that appeared with the release of 0.930 😬

Reply:
Annoying!

It looks like all these issues are in the `legacy_caching/hashing.py` file - as a stopgap, we could add `# type: ignore` to the top of that file, along with a TODO to investigate after the break. (#core-platform can be in charge of fixing this issue.)

Reply:
We got a solution with https://github.com/streamlit/streamlit/pull/4218 that fixes with the latest mypy. I've merged it in this one.